tests/security/issues/test_builder.py

```
@@ -433,19 +297,41 @@ def test_first_seen_falls_back_to_metadata_created_at() -> None:
+def test_message_with_heading_is_escaped() -> None:
+    """Markdown headings in the message field must not create real headings."""
     alert = Alert.from_dict({
-        "metadata": {"rule_id": "X"},  # no updated_at / created_at
-        "alert_details": {},
+        "metadata": {"rule_id": "CVE-TEST", "rule_description": "Test vuln"},
+        "alert_details": {"message": "## Black is the uncompromising formatter."},
         "rule_details": {},
     })
     body = build_child_issue_body(alert)
-    assert body.count("N/A") >= 2
+    assert not any(l.strip().startswith("## Black") for l in body.split("\n")), "Message heading should be escaped"
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)